backend/src/api/routes/payments/razorpay.routes.ts

```
@@ -0,0 +1,313 @@
+import { Router } from 'express';
+import type { Response, NextFunction } from 'express';
```

ESLint

[error] 'express' import is duplicated.

(no-duplicate-imports)